### DISCUSSION TITLE
help file

### DISCUSSION THREAD

**Discussion by sov44** [sumatrapdfreader/sumatrapdf#4228] [Documentation]
Please add a localized help file to the installer.

**Comment by GitHubRulesOK** [comment]
In the past there was a manual to language converter but the problems  of so many languages 
exceeded the simplicity of just read the manual with a browsers page translator, here in French  
https://www.sumatrapdfreader.org/docs/Installer-cmd-line-arguments
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/c8a49f05-f2ae-4369-97a6-c65e888e1342)

**Comment by sov44** [comment]
No need for a converter. The English help file sumatra.chm is sufficient https://www.sumatrapdfreader.org/docs/SumatraPDF-documentation
Users will translate it into their native language.

**Reply by GitHubRulesOK** [reply]
AFAIK there never should be a CHM it is almost redundant MS insecure format (one the very few places SumatraPDF is more vulnerable)
If you have a sumatrapdf.chm it should only be of your own compilation. as it is known 8 years ago it can be self infecting.

**Comment by kjk** [comment]
I've opened https://github.com/sumatrapdfreader/sumatrapdf/issues/4234 to track this

**Comment by Kcohen202** [comment]
I have an easy Idea, make a downloadable pdf. in english, everyone can open the pdf in their browser. have them download the google translate extension installed in their browser it should translate the pdf into their language, google is constantly translating new languages back and forth, there isnt much written languages left they haven't catalogued now they're working on spoken. You can also go to their website and have them translate the document and popout a tranlated one as well so either way, downloadable pdf is the way to go. 
no reason to intigrate help files, Microsoft doesn't even do that anymore.

**Comment by GitHubRulesOK** [comment]
@Kcohen202 
I agree the help file is not auto translated by Microsoft they say English to english since thats my Locale however for others it should be translated to own language, without changing to google, Edge should work.
![image](https://github.com/user-attachments/assets/72f8a5d8-ff58-42bf-a7b4-d92218f15e6b)
![image](https://github.com/user-attachments/assets/4493613f-85c5-4d73-afb8-7d4e72f8e0fa)
![image](https://github.com/user-attachments/assets/bf072098-c429-46aa-92d9-47e4ef33c6e7)
